### PR TITLE
Blocked and unblocked users

### PR DESCRIPTION
**Pull request by ArvindKumarSuroju**

### CODE REVIEW

**Comment by jatinparmar96** [comment]
Is the feature finished? @ArvindKumarSuroju

File: src/components/screens/myAmigoes/EmergencyScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            placeholder="Please tell me what happened"
+            _dark={{
+              placeholderTextColor: 'gray.300',
+            }}
+            mb="5"
+          />
+        </Box>
+        <Box backgroundColor="#FFE4E0" mb={5}>
+          <Text variant="body" padding={4} ml={4}>
+            amigoes will contact this person and their emergency contact if
```
Comment (jatinparmar96) [inline src/components/screens/myAmigoes/EmergencyScreen.tsx:79]: Change to translated text here

File: src/components/screens/myAmigoes/EmergencyScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            <VStack space={3}>
+              <Text
+                variant="h3"
+                color="#3FA8AE"
+                ml={5}
+                mr={5}
+                padding={3}
+                textAlign="center"
+              >
+                Are you sure to send emergency?
```
Comment (jatinparmar96) [inline src/components/screens/myAmigoes/EmergencyScreen.tsx:136]: Same here

File: package.json [hunk 3]
```diff
@@ -38,6 +38,7 @@
     "firebase": "^8.2.3",
     "geolib": "^3.3.3",
     "i18n-js": "^4.1.1",
+    "moment": "^2.29.4",
```
Comment (jatinparmar96) [inline package.json:41]: Moment has been depreceted

**Review by jatinparmar96 — CHANGES_REQUESTED** [review]

File: src/components/screens/myAmigoes/EmergencyScreen.tsx (hunk 1 above)
Comment (ArvindKumarSuroju) [reply src/components/screens/myAmigoes/EmergencyScreen.tsx:79]: I have now added the translated text for the fixed strings, and let's check for the data retrieved from the database after the beta demo.

File: package.json (hunk 3 above)
Comment (ArvindKumarSuroju) [reply package.json:41]: I was using locale String, but I could not get the exact format used for the date in the mockup. So I have used moment for the date for conversion.

**Pull request merged**